opentok/opentok-meet
# [OPENTOK-47890] Updating dep to remove custom TURN servers

#### What is this PR doing?
Updating dependency which removes the custom STUN/TURN config.
https://github.com/michaelwhittemore/OpenTok-Angular/commit/6d2e6d21f56dca78c562a1019284c433c09e5658

#### How should this be manually tested?
Run meet locally and check P2P sessions work. Please use ngrok to make sure you are accessing via no localhost

#### What are the relevant tickets?
OPENTOK-47890
